salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -16,6 +18,13 @@ Correct saltenv "{{ saltenv }}" for downgrade to "{{ dest_version }}":
 
 {%- endif %}
 
+{%- if not defaults.downgrade_supported and not pillar.force_downgrade | default(False) %}
+
+Downgrade from current version is not supported:
+  test.fail_without_changes
```

Comment:
To be clear I would add a clear comment here something like "etcd downgrade not supported blablabla" + Maybe a link to a github issue for this https://github.com/scality/metalk8s/issues/1750 (maybe we want to do some cleaning in the ticket (and talk about 2.6 instead of non-released 2.3) but no related to this PR)

Reply:
I'd rather link to 2.6.0 release notes, where we will explain this limitation, but agreed, additional help would be very useful for users.